packages/nimble-tokens/CONTRIBUTING.md

```diff
@@ -28,6 +28,18 @@ Tokens are generated using the [Style Dictionary](https://amzn.github.io/style-d
 
 ## Updating icons
 
+### Icon naming
+
+Prefer [FontAwesome](https://fontawesome.com) names for icons. I.e. Minimize the use of metaphors, and choose descriptive and unambiguous names. 
+Add all appropriate metaphors/synonyms to the `icon-metadata.ts` file, so we can search for icons in Storybook via names or metaphors.
+
+E.g. 
+| Descriptive name       | Poor name              |
```

Comment:
Could you make it more clear that the poor names are great synonyms/metaphors? Maybe either by updating that column heading or by adding a new column for synonyms?